dev/com.ibm.ws.microprofile.rest.client_fat/fat/src/com/ibm/ws/microprofile/rest/client/fat/BasicCdiTest.java

```diff
@@ -74,14 +76,16 @@ public class BasicCdiTest extends FATServletClient {
     @BeforeClass
     public static void setUp() throws Exception {
         ShrinkHelper.defaultDropinApp(remoteAppServer, "basicRemoteApp", new DeployOptions[] {DeployOptions.OVERWRITE}, "remoteApp.basic");
-        if (JakartaEE9Action.isActive()) {
+        if (JakartaEE9Action.isActive() | JakartaEE10Action.isActive()) {
             remoteAppServer.changeFeatures(Arrays.asList("componenttest-2.0", "restfulWS-3.0", "ssl-1.0", "jsonb-2.0"));
```

Comment:
This doesn't matter because `remoteAppServer` isn't the server under test, it's just the target for `mpRestClient` to make requests against.

Note that `RepeatTests` doesn't update the features for `mpRestClient10.remoteServer`, only for `mpRestClient10.basic.cdi`. We wouldn't need to change the features for this server at all, except that `ShrinkHelper` will automatically transform apps when we're doing an EE9 or EE10 repeat, so the remote server does need to have EE9 compatible features.